src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/AbsentReasonsController.kt

```diff
@@ -24,23 +23,17 @@ import uk.gov.justice.digital.hmpps.whereabouts.model.AbsentSubReason.Companion.
 class AbsentReasonsController {
 
   @GetMapping
-  fun reasons(): AbsentReasonsDto =
-    AbsentReasonsDto(
-      paidReasons,
-      unpaidReasons,
-      iepTriggers,
-    )
+  fun reasons() = AbsentReasonsDto(
+    paidReasons.toReasonDto(),
+    unpaidReasons.toReasonDto(),
+    iepTriggers.toList(),
+    absentSubReasonTriggers,
+    paidSubReasons.toSubReasonDto(),
+    unpaidSubReasons.toSubReasonDto(),
+  )
 
   @GetMapping(value = ["/v2"])
-  fun reasonsV2(): AbsentReasonsV2Dto =
-    AbsentReasonsV2Dto(
-      paidReasons.toReasonDto(),
-      unpaidReasons.toReasonDto(),
-      iepTriggers.toList(),
-      absentSubReasonTriggers,
-      paidSubReasons.toSubReasonDto(),
-      unpaidSubReasons.toSubReasonDto(),
-    )
+  fun reasonsV2() = reasons()
```

Comment:
Old `reasons()` version no longer used.  Will get front end to switch back from `/v2` version and then can remove `/v2` later on